src/openforms/forms/tests/test_restore_version.py

```diff
@@ -294,3 +298,204 @@ def test_restore_form_with_reusable_form_definition(self):
             },
         )
         self.assertFalse(form_steps[1].form_definition.is_reusable)
+
+
+FORM_STEP = [
+    {
+        "form": "http://testserver/api/v1/forms/324cadce-a627-4e3f-b117-37ca232f16b2",
+        "form_definition": "http://testserver/api/v1/form-definitions/f0dad93b-333b-49af-868b-a6bcb94fa1b8",
+        "index": 0,
+        "slug": "icecream-form-step",
+        "uuid": "3ca01601-cd20-4746-bce5-baab47636823",
+    }
+]
+
+FORM = [
+    {
+        "active": True,
+        "authentication_backends": [],
+        "is_deleted": False,
+        "login_required": False,
+        "maintenance_mode": False,
+        "name": "Icecream questionnaire",
+        "internal_name": "Icecream questionnaire",
+        "product": None,
+        "show_progress_indicator": True,
+        "slug": "icecream-questionnaire",
+        "url": "http://testserver/api/v1/forms/324cadce-a627-4e3f-b117-37ca232f16b2",
+        "uuid": "324cadce-a627-4e3f-b117-37ca232f16b2",
+    }
+]
+
+FORM_DEFINITION = [
+    {
+        "configuration": {
+            "components": [{"key": "favouriteFlavour", "type": "textfield"}]
+        },
+        "name": "Icecream questions",
+        "internal_name": "Icecream questions",
+        "slug": "icecream-questions",
+        "url": "http://testserver/api/v1/form-definitions/f0dad93b-333b-49af-868b-a6bcb94fa1b8",
+        "uuid": "f0dad93b-333b-49af-868b-a6bcb94fa1b8",
+    }
+]
+
+FORM_VARIABLES = [
+    {
+        "form": "http://testserver/api/v1/forms/324cadce-a627-4e3f-b117-37ca232f16b2",
+        "name": "CO2 footprint",
+        "key": "co2-footprint",
+        "source": FormVariableSources.user_defined,
+        "data_type": FormVariableDataTypes.float,
+    }
+]
+
+
+class RestoreVersionsWithVariablesTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.export_blob = {
+            "forms": json.dumps(FORM),
+            "formDefinitions": json.dumps(FORM_DEFINITION),
+            "formSteps": json.dumps(FORM_STEP),
+            "formLogic": json.dumps([]),
+        }
```

Comment:
make sure to call the `super` method. It's really annoying if we have to add mixins to existing test cases and not having them working because of the missing super call.